samples/cpp/benchmark_app/main.cpp

```diff
@@ -1028,7 +1047,9 @@ int main(int argc, char* argv[]) {
             ++iteration;
 
             execTime = std::chrono::duration_cast<ns>(Time::now() - startTime).count();
-            processedFramesN += batchSize;
+            if (inside_measurement_window) {
+                processedFramesN += batchSize;
```

Comment:
How it would work for async api where FPS is calculated as `1000.0 * processedFramesN / totalDuration`? We still have the same duration but reduced number of processed frames. 

BTW could you explain a little bit why this feature was requested?

Reply:
Feature not required